standalone/src/main/java/io/pravega/local/LocalPravegaEmulator.java

```diff
@@ -125,12 +126,11 @@ public void run() {
 
             log.info("Starting Pravega Emulator with ports: ZK port {}, controllerPort {}, SegmentStorePort {}",
                     conf.getZkPort(), conf.getControllerPort(), conf.getSegmentStorePort());
-
             localPravega.start();
-
-            System.out.println(
-                    String.format("Pravega Sandbox is running locally now. You could access it at %s:%d", "127.0.0.1",
-                            conf.getControllerPort()));
+            log.info("");
+            log.info("Pravega Sandbox is running locally now. You could access it at {}:{}.", "127.0.0.1", conf.getControllerPort());
+            log.info("For more detailed logs, see: {}", String.format("%s/%s", System.getProperty("user.dir"), "standalone/standalone.log"));
```

Comment:
You don't need to do String format here. the `log.info` arg is a formatted string. Use `{}` instead of `%s`.